[
    "dc7o5wtbfqu64d3ld/include/python3.8/cpython/object.h:260:30: note:\n            declared here\n     522      260 |     Py_DEPRECATED(3.8) int (*tp_print)(PyObject *, FILE *,\n            int);\n     523          |                              ^~~~~~~~\n     524    ext/_yaml.c: In function '__Pyx_ExceptionSave':\n>> 525    ext/_yaml.c:20800:21: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     526    20800 |     *type = tstate->exc_type;\n     527          |                     ^~~~~~~~\n     528          |                     curexc_type\n  >> 529    ext/_yaml.c:20801:22: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?",
    "524    ext/_yaml.c: In function '__Pyx_ExceptionSave':\n  >> 525    ext/_yaml.c:20800:21: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     526    20800 |     *type = tstate->exc_type;\n     527          |                     ^~~~~~~~\n     528          |                     curexc_type\n>> 529    ext/_yaml.c:20801:22: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     530    20801 |     *value = tstate->exc_value;\n     531          |                      ^~~~~~~~~\n     532          |                      curexc_value\n  >> 533    ext/_yaml.c:20802:19: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?",
    "528          |                     curexc_type\n  >> 529    ext/_yaml.c:20801:22: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     530    20801 |     *value = tstate->exc_value;\n     531          |                      ^~~~~~~~~\n     532          |                      curexc_value\n>> 533    ext/_yaml.c:20802:19: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?\n     534    20802 |     *tb = tstate->exc_traceback;\n     535          |                   ^~~~~~~~~~~~~\n     536          |                   curexc_traceback\n     537    ext/_yaml.c: In function '__Pyx_ExceptionReset':\n  >> 538    ext/_yaml.c:20814:24: error: 'PyThreadState' {aka 'struct _ts'} has",
    ">> 533    ext/_yaml.c:20802:19: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?\n     534    20802 |     *tb = tstate->exc_traceback;\n     535          |                   ^~~~~~~~~~~~~\n     536          |                   curexc_traceback\n     537    ext/_yaml.c: In function '__Pyx_ExceptionReset':\n>> 538    ext/_yaml.c:20814:24: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     539    20814 |     tmp_type = tstate->exc_type;\n     540          |                        ^~~~~~~~\n     541          |                        curexc_type\n  >> 542    ext/_yaml.c:20815:25: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?",
    "537    ext/_yaml.c: In function '__Pyx_ExceptionReset':\n  >> 538    ext/_yaml.c:20814:24: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     539    20814 |     tmp_type = tstate->exc_type;\n     540          |                        ^~~~~~~~\n     541          |                        curexc_type\n>> 542    ext/_yaml.c:20815:25: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     543    20815 |     tmp_value = tstate->exc_value;\n     544          |                         ^~~~~~~~~\n     545          |                         curexc_value\n  >> 546    ext/_yaml.c:20816:22: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?",
    "541          |                        curexc_type\n  >> 542    ext/_yaml.c:20815:25: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     543    20815 |     tmp_value = tstate->exc_value;\n     544          |                         ^~~~~~~~~\n     545          |                         curexc_value\n>> 546    ext/_yaml.c:20816:22: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?\n     547    20816 |     tmp_tb = tstate->exc_traceback;\n     548          |                      ^~~~~~~~~~~~~\n     549          |                      curexc_traceback\n  >> 550    ext/_yaml.c:20817:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?",
    "545          |                         curexc_value\n  >> 546    ext/_yaml.c:20816:22: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?\n     547    20816 |     tmp_tb = tstate->exc_traceback;\n     548          |                      ^~~~~~~~~~~~~\n     549          |                      curexc_traceback\n>> 550    ext/_yaml.c:20817:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     551    20817 |     tstate->exc_type = type;\n     552          |             ^~~~~~~~\n     553          |             curexc_type\n  >> 554    ext/_yaml.c:20818:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?",
    "549          |                      curexc_traceback\n  >> 550    ext/_yaml.c:20817:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     551    20817 |     tstate->exc_type = type;\n     552          |             ^~~~~~~~\n     553          |             curexc_type\n>> 554    ext/_yaml.c:20818:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     555    20818 |     tstate->exc_value = value;\n     556          |             ^~~~~~~~~\n     557          |             curexc_value\n  >> 558    ext/_yaml.c:20819:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?",
    "553          |             curexc_type\n  >> 554    ext/_yaml.c:20818:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     555    20818 |     tstate->exc_value = value;\n     556          |             ^~~~~~~~~\n     557          |             curexc_value\n>> 558    ext/_yaml.c:20819:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?\n     559    20819 |     tstate->exc_traceback = tb;\n     560          |             ^~~~~~~~~~~~~\n     561          |             curexc_traceback\n     562    ext/_yaml.c: In function '__Pyx_GetException':\n  >> 563    ext/_yaml.c:20862:24: error: 'PyThreadState' {aka 'struct _ts'} has",
    ">> 558    ext/_yaml.c:20819:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?\n     559    20819 |     tstate->exc_traceback = tb;\n     560          |             ^~~~~~~~~~~~~\n     561          |             curexc_traceback\n     562    ext/_yaml.c: In function '__Pyx_GetException':\n>> 563    ext/_yaml.c:20862:24: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     564    20862 |     tmp_type = tstate->exc_type;\n     565          |                        ^~~~~~~~\n     566          |                        curexc_type\n  >> 567    ext/_yaml.c:20863:25: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?",
    "562    ext/_yaml.c: In function '__Pyx_GetException':\n  >> 563    ext/_yaml.c:20862:24: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     564    20862 |     tmp_type = tstate->exc_type;\n     565          |                        ^~~~~~~~\n     566          |                        curexc_type\n>> 567    ext/_yaml.c:20863:25: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     568    20863 |     tmp_value = tstate->exc_value;\n     569          |                         ^~~~~~~~~\n     570          |                         curexc_value\n  >> 571    ext/_yaml.c:20864:22: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?",
    "566          |                        curexc_type\n  >> 567    ext/_yaml.c:20863:25: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     568    20863 |     tmp_value = tstate->exc_value;\n     569          |                         ^~~~~~~~~\n     570          |                         curexc_value\n>> 571    ext/_yaml.c:20864:22: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?\n     572    20864 |     tmp_tb = tstate->exc_traceback;\n     573          |                      ^~~~~~~~~~~~~\n     574          |                      curexc_traceback\n  >> 575    ext/_yaml.c:20865:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?",
    "570          |                         curexc_value\n  >> 571    ext/_yaml.c:20864:22: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?\n     572    20864 |     tmp_tb = tstate->exc_traceback;\n     573          |                      ^~~~~~~~~~~~~\n     574          |                      curexc_traceback\n>> 575    ext/_yaml.c:20865:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     576    20865 |     tstate->exc_type = local_type;\n     577          |             ^~~~~~~~\n     578          |             curexc_type\n  >> 579    ext/_yaml.c:20866:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?",
    "574          |                      curexc_traceback\n  >> 575    ext/_yaml.c:20865:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_type'; did you mean 'curexc_type'?\n     576    20865 |     tstate->exc_type = local_type;\n     577          |             ^~~~~~~~\n     578          |             curexc_type\n>> 579    ext/_yaml.c:20866:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     580    20866 |     tstate->exc_value = local_value;\n     581          |             ^~~~~~~~~\n     582          |             curexc_value\n  >> 583    ext/_yaml.c:20867:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?",
    "578          |             curexc_type\n  >> 579    ext/_yaml.c:20866:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_value'; did you mean 'curexc_value'?\n     580    20866 |     tstate->exc_value = local_value;\n     581          |             ^~~~~~~~~\n     582          |             curexc_value\n>> 583    ext/_yaml.c:20867:13: error: 'PyThreadState' {aka 'struct _ts'} has\n             no member named 'exc_traceback'; did you mean 'curexc_traceback'?\n     584    20867 |     tstate->exc_traceback = local_tb;\n     585          |             ^~~~~~~~~~~~~\n     586          |             curexc_traceback\n  >> 587    error: command '/p/vast1/koparasy/BUILD/head/spack/lib/spack/env/gc\n            c/gcc' failed with exit status 1"
]